src/web/src/pages/Main.vue

```diff
@@ -290,9 +290,10 @@ export default {
             exclusive_date_end.setDate(course.date_end.getDate() + 1);
             semester = session.semester;
             calendarBuilder.addEvent(
-              `Class: ${course.title}`,
+              `Class: ${course.full_title ? course.full_title : course.title}`,
```

Comment:
what about `course.full_title || course.title` :)

Reply:
that makes sense.